src/cli/initPolykey.ts

```diff
@@ -5,7 +5,7 @@ import inquirer from 'inquirer'
 import Configstore from 'configstore'
 import PolyKey from '../lib/Polykey'
 import cliProgress from 'cli-progress'
-import KeyManager from '@polykey/KeyManager'
+import KeyManager from '@polykey/keys/KeyManager'
 import { actionRunner, pkLogger, PKMessageType } from './polykey'
 
```

Comment:
I've added in a new key value pair for polykey cli.

The `bin/pk.js` file was generated by hand, I am not sure how to get it to be generated by webpack. I can set it as an entry and then have it compiled but that would compile the rest of the code into that file as well since webpack follows `require` imports.

Perhaps we should also add another key for our alias `pk`. I've added it in for now, otherwise we can make that something that happens with our desktop app installation process.

Reply:
Yea that's fine to generate by hand.

Reply:
awesome, I shall resolve this then :+1: